ui/src/app/utils/runtime-presets.ts

```diff
@@ -18,7 +22,7 @@ export const runtimePresets: {
       // TODO: Support specifying toolDockerImage here.
       gceConfig: {
         machineType: 'n1-standard-4',
-        diskSize: 100,
+        diskSize: MIN_DISK_SIZE_GB,
```

Comment:
this is a nice change, thanks.  What about a const for `n1-standard-4` as well?

Reply:
Done - actually we already had this constant.